docroot/themes/humsci/humsci_basic/src/scss/components/_buttons.scss

```diff
@@ -13,7 +13,7 @@ a.button {
     min-height: auto;
 
     @include hb-colorful {
-      @include hb-pairing-color('color', 'secondary');
+      @include hb-pairing-color('color', 'secondary', $important: true);
```

Comment:
please avoid any `important` in  css if at all possible

Reply:
Hm. Our mixins don't use !important by default, but maybe this was a special case?

@kaseybon, I'm seeing that this change was made [here in our QA PR](https://github.com/SU-HSDO/suhumsci/pull/571/files#diff-b79f5db116264e3c9c0f5657e44f6fa2R16). Do you remember the reasoning behind this, and whether there's another way to get the specificity we need without !important?

Reply:
We do try to avoid `!important` whenever possible but there are a couple of scenarios where we do need it. We need to use `!important` when styling buttons in the dark footer variant which causes a bit of a chain reaction with the other buttons which is why the reset button here is getting `$important: true`.

Reply:
is it possible to use greater specification of the css selector instead of using important?

Reply:
@pookmish, Kasey is out today. I assume she explored some of those options, but I'm not sure. I've got some meetings but I could take a pass at this later today if you want.

Reply:
yeah i'd just like to see if its possible to avoid. typically its only needed when overriding styles set with javascript. this is the only thing holding up the PR at this point.

Reply:
@pookmish, I did some reshuffling and was able to get this working without `!important` in ddf5a75. How does that look?

Reply:
Cool thanks for that